deploy/0062_deploy_gems_catalysts_registry.ts

```diff
@@ -5,11 +5,11 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const {deployments, getNamedAccounts} = hre;
   const {deploy} = deployments;
 
-  const {gemsCatalystsRegistryOwner} = await getNamedAccounts();
+  const {gemsCatalystsRegistryAdmin, deployer} = await getNamedAccounts();
   await deploy(`GemsCatalystsRegistry`, {
-    from: gemsCatalystsRegistryOwner,
+    from: deployer,
     log: true,
-    args: [gemsCatalystsRegistryOwner],
+    args: [gemsCatalystsRegistryAdmin],
```

Comment:
we usually set admin to be first `deployer` so it can start adding the gems and catalyst
and then change the admin in another deploy script, see how Sand, Asset, Land are setup